extension/app/ts/background/popupMessageHandlers.ts

```diff
@@ -211,3 +211,20 @@ export async function getAddressBookData(_simulator: Simulator, payload: PopupMe
 		}
 	})
 }
+
+export async function openAddressBook(_simulator: Simulator, _payload: PopupMessage) {
+	const html = '/html/addressBook.html'
+
+	const openInNewTab = async () => {
+		const tab = await browser.tabs.create({ url: `..${ html }` })
```

Comment:
```suggestion
	const openInNewTab = async () => {
		const tab = await browser.tabs.create({ url: '../html/addressBook.htm' })
```
Whats the reason for setting the URI from ``'...' + html``?

Reply:
yeah, that's interesting. I doesn't do anything, I removed it